.changeset/beige-months-obey.md

```diff
@@ -0,0 +1,5 @@
+---
+'@sveltejs/kit': patch
+---
+
+Add disableScrollHandling function
```

Comment:
Since this is a breaking change, perhaps we can add some information here on how to migrate over, and a big `BREAKING CHANGE` text. I noticed on discord many still have questions regarding the recent breaking changes, even though we have provided some helpful error messages.